packages/react/src/components/Authenticator/Authenticator.tsx

```diff
@@ -109,6 +116,7 @@ export function AuthenticatorInternal({
  * [📖 Docs](https://ui.docs.amplify.aws/react/connected-components/authenticator)
  */
 export function Authenticator(props: AuthenticatorProps): JSX.Element {
+  appendToCognitoUserAgent(`@aws-amplify/ui-react/${VERSION}`);
```

Comment:
Thinking we will want to wrap this in a `useEffect` with an empty dependency array to ensure it doesn't run on each render